manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-core-operators
+  name: openshift-cluster-openshift-controller-manager-operator-builder-deployer-config
```

Comment:
ah, you learn names from the best.  @liggitt @sttts 

In this case though, how about `openshift-controller-manager-images`?